### PR TITLE
generator markdown and translate

### PR DESCRIPTION
**Pull request by jueduizone**
generator markdown and translate

### CODE REVIEW

File: script/youdao_translator_api.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
```
Comment (WillemJiang) [inline script/youdao_translator_api.py:1]: Not sure if this code is wrote by you.
Could you add a license header for this file if you wrote it ?

**Comment by jueduizone** [comment]
fixed, code from official user guide, add some description

**Review by WillemJiang — APPROVED** [review]

**Comment by WillemJiang** [comment]
@jueduizone，as there are bunch of code you committed,  you need to fill the iCLA first.
You can find more information here https://www.apache.org/licenses/contributor-agreements.html#clas

**Review by WillemJiang — APPROVED** [review]

**Comment by jueduizone** [comment]
fixed the bug of image style and session-type

**Review by WillemJiang — APPROVED** [review]

**Pull request merged**